frontend/src/app/todo/todo.service.ts

```diff
@@ -1,32 +1,15 @@
 import { Injectable } from '@angular/core';
-import { HttpClient } from '@angular/common/http';
 import { environment } from '../../environments/environment';
 import { Todo } from './todo.intercace';
-import { Observable } from 'rxjs';
+import { ApiRoute, ApiUrl, CrudService } from 'ngx-crud';
 
-@Injectable({
+@Injectable(
+{
 	providedIn: 'root'
 })
-export class TodoService
+@ApiUrl(environment.apiUrl)
+@ApiRoute(environment.apiRoutes.todos)
+export class TodoService extends CrudService<Todo, Todo, Todo, Todo, Todo, Todo[], Todo, Todo, Todo, Todo, void>
```

Comment:
You did not merge develop branch into that one - this is false commit and we are using crud service custom !!!